### PR TITLE
[ENG-1113] Data pipeline example

### PR DESCRIPTION
**Pull request by tebeka**

### CODE REVIEW

File: data_pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+In this example, we'll use [ngrok](https://ngrok.com/)
+
+```
+ngrok http 9980
+```
+
+You'll see a screen with a line like:
+
+```
+Forwarding                    https://e7c499cae8d9.ngrok.app -> http://localhost:9980
```
Comment (daabr) [inline data_pipeline/README.md:34]: Extreme nit: even though the spaces are accurate, they're not serving a useful purpose here.

```suggestion
Forwarding https://e7c499cae8d9.ngrok.app -> http://localhost:9980
```

File: data_pipeline/pipeline.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+@autokitteh.activity
+def get_s3_object(bucket, key):
+    s3_client = boto3.client(
+        "s3",
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_KEY,
+        region_name=AWS_REGION,
+    )
```
Comment (daabr) [inline data_pipeline/pipeline.py:55]: Not critical, but FYI feel free to use: https://github.com/autokitteh/autokitteh/blob/main/runtimes/pythonrt/py-sdk/autokitteh/aws.py

**Review by daabr — APPROVED** [review]
👍 I'm assuming you're going to remove the original from the samples repo?

File: data_pipeline/README.md (hunk 1 above)
Comment (tebeka) [reply data_pipeline/README.md:34]: I prefer to show output that is as similar to the real one.

File: data_pipeline/pipeline.py (hunk 2 above)
Comment (tebeka) [reply data_pipeline/pipeline.py:55]: Good point, but testing new code will take too much of my time.

**Comment by tebeka** [comment]
Closing this one since commit isn't signed. Opened https://github.com/autokitteh/kittehub/pull/25

**Pull request closed without merging**